src/main/java/today/creame/web/influence/entrypoint/rest/InfluenceApplicationRestController.java

```diff
@@ -50,37 +49,16 @@ public ResponseEntity<Body<SimpleBodyData<Long>>> register(
     }
 
     @GetMapping("/admin/v1/influence/applications")
-    public ResponseEntity<Body<List<InfluenceApplicationResult>>> list(
-        @RequestParam(name = "statues", defaultValue = "") String statues) {
-        log.debug("request param: {}", statues);
+    public ResponseEntity<PageBody<InfluenceApplicationResult>> list(
+            @PageableDefault(sort = "id", direction = Sort.Direction.DESC, size = 10) Pageable pageable,
+            @RequestParam(name = "status", defaultValue = "") List<InfluenceApplicationStatus> status) {
```

Comment:
조건이 늘어날 경우를 생각해서  ModelAttribute 변경하면 어떨까요?!

Reply:
수정 완료

Reply:
단순히 GET API의 파라미터를 받기 위해서라면 별도 어노테이션 없이 아래처럼 사용해도 충분한 거 같습니다
```suggestion
            @PageableDefault(sort = "id", direction = Sort.Direction.DESC, size = 10) Pageable pageable,
            InfluenceApplicationSearchRequest searchRequest) {
```
Spring MVC는 별도 어노테이션이 없다면 요청 매개 변수와 객체 필드의 이름이 정확히 일치하는 경우 자동으로 바인딩 해줌
컨트롤러 어노테이션 관련 내용은 https://docs.spring.io/spring-framework/docs/current/reference/html/web.html#mvc-ann-arguments 참고!

Reply:
객체 필드 이름과 요청 매개 변수 이름이 일치하면 Spring은 해당 매개 변수를 객체 필드에 자동으로 바인딩! 바인딩시 jackson JSON 사용해서 바인딩하기 때문에 setter 없어도 됨!